### ISSUE TITLE
Checked open but cant see my issues - whenever I edit a save and load in game it takes me back to the opening "begin initialisation" screen

### ISSUE THREAD

**Issue by sjuk86** [goatfungus/NMSSaveEditor#1083]
[NMSSaveEditor.log](https://github.com/user-attachments/files/17045902/NMSSaveEditor.log)

l've tried several times, only happens when I amend the maual save in the editor as the auto save does nothing. have to run as admin otherwise get save error.

**Comment by ZataraTMM** [comment]
I'm getting the same exact thing. Trying to figure out how to rollback now. Do you know\did you fix this?

**Comment by sjuk86** [comment]
No mate I wish I did! I’ve ended up restarting like 5 times now.Tried opening a backup in the editor and saving but that’s not working. Also extracted a backup zip to copy into the save location but it doesn’t give folder to add so unsure where to copy/overwrite or the naming convention to create a new folder. Let me know if you figure anything Sent from my iPhoneOn 18 Sep 2024, at 21:58, ZataraTMM ***@***.***> wrote:﻿
I'm getting the same exact thing. Trying to figure out how to rollback now. Do you know\did you fix this?

**Comment by ZataraTMM** [comment]
> No mate I wish I did! I’ve ended up restarting like 5 times now.Tried opening a backup in the editor and saving but that’s not working. Also extracted a backup zip to copy into the save location but it doesn’t give folder to add so unsure where to copy/overwrite or the naming convention to create a new folder. Let me know if you figure anything Sent from my iPhoneOn 18 Sep 2024, at 21:58, ZataraTMM ***@***.***> wrote:﻿
> I'm getting the same exact thing. Trying to figure out how to rollback now. Do you know\did you fix this?
> 
> —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

I managed to fix it once. Rn i went to bed I'll try again tomorrow. Was pretty pissed too as the save looks normal aswell. I'll see what i can do

**Comment by sjuk86** [comment]
Yeah the save looks right in game as time matches with the last saved in the editor but just won’t loadSent from my iPhoneOn 18 Sep 2024, at 22:10, ZataraTMM ***@***.***> wrote:﻿

No mate I wish I did! I’ve ended up restarting like 5 times now.Tried opening a backup in the editor and saving but that’s not working. Also extracted a backup zip to copy into the save location but it doesn’t give folder to add so unsure where to copy/overwrite or the naming convention to create a new folder. Let me know if you figure anything Sent from my iPhoneOn 18 Sep 2024, at 21:58, ZataraTMM @.***> wrote:﻿
I'm getting the same exact thing. Trying to figure out how to rollback now. Do you know\did you fix this?
—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: @.***>

I managed to fix it once. Rn i went to bed I'll try again tomorrow. Was pretty pissed too as the save looks normal aswell. I'll see what i can do

**Comment by BlueKAtrc** [comment]
@sjuk86 you have the Gamepass version of the game; @ZataraTMM which version of the game do you have?

**Comment by ZataraTMM** [comment]
> @sjuk86 you have the Gamepass version of the game; @ZataraTMM which version of the game do you have?

Yeah i have the gamepass version. Realized\read few things that people are having probs

**Comment by sjuk86** [comment]
> @sjuk86 you have the Gamepass version of the game; @ZataraTMM which version of the game do you have?

Yeah gamepass

**Comment by ZataraTMM** [comment]
> @sjuk86 you have the Gamepass version of the game; @ZataraTMM which version of the game do you have?

i kept trying to bounce between NMS and Nom to try to fix this, hoping any would work as i've used both. i have the save files or auto backups  and i vaugley remember when it broke around what time. but i have no idea how to use them, or atleast do them manually as both of them crash when trying to pick a backup, i tought manually. Is there an actual way for this? tried renaming folders/etc fkin around, nothing.

**Comment by sjuk86** [comment]
> > @sjuk86 you have the Gamepass version of the game; @ZataraTMM which version of the game do you have?
> 
> 
> 
> i kept trying to bounce between NMS and Nom to try to fix this, hoping any would work as i've used both. i have the save files or auto backups  and i vaugley remember when it broke around what time. but i have no idea how to use them, or atleast do them manually as both of them crash when trying to pick a backup, i tought manually. Is there an actual way for this? tried renaming folders/etc fkin around, nothing.

I know where the folders are for the gamepass version but extracting a backup from this only gives the 3 files and not the folder itself- so them comparing to the contents of the file locations for the game don't match what was extracted.

Also why doesn't the save editor work by just loading a backup up in the slot?

**Comment by ZataraTMM** [comment]
> > > @sjuk86 you have the Gamepass version of the game; @ZataraTMM which version of the game do you have?
> > 
> > 
> > 
> > i kept trying to bounce between NMS and Nom to try to fix this, hoping any would work as i've used both. i have the save files or auto backups  and i vaugley remember when it broke around what time. but i have no idea how to use them, or atleast do them manually as both of them crash when trying to pick a backup, i tought manually. Is there an actual way for this? tried renaming folders/etc fkin around, nothing.
> 
> I know where the folders are for the gamepass version but extracting a backup from this only gives the 3 files and not the folder itself- so them comparing to the contents of the file locations for the game don't match what was extracted.
> 
> Also why doesn't the save editor work by just loading a backup up in the slot? 

I just spent a hr or two on discord with someone and they helped me out. But in short. I exported my new save json and loaded it in the latest working backup i had which was 2 days old.

**Comment by sjuk86** [comment]
> > > > @sjuk86 you have the Gamepass version of the game; @ZataraTMM which version of the game do you have?
> > > 
> > > 
> > > 
> > > i kept trying to bounce between NMS and Nom to try to fix this, hoping any would work as i've used both. i have the save files or auto backups  and i vaugley remember when it broke around what time. but i have no idea how to use them, or atleast do them manually as both of them crash when trying to pick a backup, i tought manually. Is there an actual way for this? tried renaming folders/etc fkin around, nothing.
> > 
> > I know where the folders are for the gamepass version but extracting a backup from this only gives the 3 files and not the folder itself- so them comparing to the contents of the file locations for the game don't match what was extracted.
> > 
> > Also why doesn't the save editor work by just loading a backup up in the slot? 
> 
> I just spent a hr or two on discord with someone and they helped me out. But in short. I exported my new save json and loaded it in the latest working backup i had which was 2 days old. 

I don't get it, can you explain more in depth please?

**Comment by ZataraTMM** [comment]
> > > > > @sjuk86 you have the Gamepass version of the game; @ZataraTMM which version of the game do you have?
> > > > 
> > > > 
> > > > 
> > > > i kept trying to bounce between NMS and Nom to try to fix this, hoping any would work as i've used both. i have the save files or auto backups  and i vaugley remember when it broke around what time. but i have no idea how to use them, or atleast do them manually as both of them crash when trying to pick a backup, i tought manually. Is there an actual way for this? tried renaming folders/etc fkin around, nothing.
> > > 
> > > I know where the folders are for the gamepass version but extracting a backup from this only gives the 3 files and not the folder itself- so them comparing to the contents of the file locations for the game don't match what was extracted.
> > > 
> > > Also why doesn't the save editor work by just loading a backup up in the slot? 
> > 
> > I just spent a hr or two on discord with someone and they helped me out. But in short. I exported my new save json and loaded it in the latest working backup i had which was 2 days old. 
> 
> I don't get it, can you explain more in depth please?

My bad. I used nomnom to fix it. I loaded in the newest backup\save i haf that works. I exported the json file from my latest one and imported it in the save that works

**Comment by ZataraTMM** [comment]
If you need more help. Join official disc and
 https://discord.com/channels/146101654981312513/964865106334875698/1286412227431366720 

Read through the latest parts as i kinda fucked up at the start then realized.

**Comment by yeetmageets** [comment]
ZataraTMM thank you I got mine fixed as well using NomNom except instead of loading a backup I just added one nanite to the messed up save and just saved and then it loaded up in game.

**Comment by sjuk86** [comment]
Nomnom doesn't give me the option to transfer the save, it's greyed out?

**Comment by ZataraTMM** [comment]
I realized my case is different because I used nom and nmsse so i have backups from both. Use what you usually used and just switch the last save json to the "latest working backup" and save, the gray area i had it too, if you do have backups with nom find one that doesn't have, those somehow corrupted and were missing some stuff from the json
